### PR TITLE
Compute Fleet SDK release for 2024-05-01-preview

### PR DESCRIPTION
**Pull request by SahilArora92**
First release for compute Fleet SDK.
Release request: https://github.com/Azure/sdk-release-request/issues/5399
Namespace review done: https://github.com/Azure/azure-sdk-pr/issues/1929
Rest API specs: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/azurefleet/resource-manager/readme.md 
Readme Tag: package-preview-2024-05

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by SahilArora92** [comment]
hi @tomsft @KrzysztofCwalina, can you please have a look at these changes. All the checks pass.

**Review by jsquire — DISMISSED** [review]

**Comment by SahilArora92** [comment]
Hi @jsquire, I am still seeing CODEOWNERS issue. Do you know how can the new labels be added for all sdks? I see they oly exist for dot net sdk.
image

**Comment by jsquire** [comment]
> Hi @jsquire, I am still seeing CODEOWNERS issue. Do you know how can the new labels be added for all sdks? I see "Compute - Fleet" only exist for [dot net sdk](https://github.com/Azure/azure-sdk-for-net/labels?q=compute+-+fleet). 

It takes 24 hours for things to sync across repositories and for permissions to sync.   You should be good by tomorrow.

**Review by ArthurMa1978 — CHANGES_REQUESTED** [review]
need switch to tsp instead of swagger spec

**Comment by SahilArora92** [comment]
> need switch to tsp instead of swagger spec

Where do i need to switch? Fleet already has [typespec](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/azurefleet) and I used the readme file to generate the sdk.

**Review by ArthurMa1978 — APPROVED** [review]

**Review by ArthurMa1978 — APPROVED** [review]

File: sdk/computefleet/Azure.ResourceManager.ComputeFleet/assets.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "net",
+  "TagPrefix": "net/computefleet/Azure.ResourceManager.ComputeFleet",
+  "Tag": ""
```
Comment (SahilArora92) [inline sdk/computefleet/Azure.ResourceManager.ComputeFleet/assets.json:5]: Hi @ArthurMa1978, Do I need a tag here as well for ComputeFleet?

Comment (ArthurMa1978) [reply sdk/computefleet/Azure.ResourceManager.ComputeFleet/assets.json:5]: Follow this guide https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md to add tests, once push test record this tag will be updated.

**Pull request merged**